src/test/java/com/cmlteam/util/UtilTest.java

```diff
@@ -0,0 +1,109 @@
+package com.cmlteam.util;
+
+import junit.framework.Assert;
+import junit.framework.TestCase;
+
+import java.util.regex.Pattern;
+
+public class UtilTest extends TestCase {
+  public void testSafeToBoolean() {
+    Assert.assertTrue(Util.safeToBoolean(true));
+    Assert.assertTrue(Util.safeToBoolean("true"));
+    Assert.assertFalse(Util.safeToBoolean(false));
+    Assert.assertFalse(Util.safeToBoolean(null));
+    Assert.assertFalse(Util.safeToBoolean("TRUE"));
+    Assert.assertFalse(Util.safeToBoolean(new Object()));
```

Comment:
For conciseness please use static import for all Assert.assert* methods.

Reply:
done